AnimateQuotes.js

```diff
@@ -0,0 +1,91 @@
+import React, { useRef, useEffect, useState } from 'react';
```

Comment:
Added file to serve as a wrapper for Quotes component in order to deliver animation functionality.